dll/shellext/netshell/lanstatusui.cpp

```diff
@@ -826,6 +827,144 @@ ShowStatusPropertyDialog(
     }
 }
 
+VOID ShowNetworkIconContextMenu(
+    _In_ HWND hwndOwner,
+    _In_ LANSTATUSUI_CONTEXT *pContext)
+{
+    if (!pContext || !pContext->pNet)
+        return;
+
+    HMENU hMenu = CreatePopupMenu();
+    if (!hMenu) return;
+
+    POINT pt;
+    GetCursorPos(&pt);
+
+    SetForegroundWindow(hwndOwner);
+
+    // The context menu items, set to their default values.
+    struct
+    {
+        UINT uID;
+        UINT uFlags;
+        UINT_PTR uIDNewItem; // wCmdID
+    } MenuItems[] =
+    {
+        {IDS_NET_ACTIVATE, MF_STRING | MF_GRAYED, IDM_NETICON_ENABLE},
+        {IDS_NET_STATUS, MF_STRING | MF_GRAYED, IDM_NETICON_STATUS},
+        {IDS_NET_REPAIR, MF_STRING | MF_GRAYED, IDM_NETICON_REPAIR},
+        {-1, 0, -1}, // Separator
```

Comment:
Compilers are yapping because of using '-1' for these unsigned fields, so let me change the proposal to this
```suggestion
        {UINT_MAX, 0, 0}, // Separator
```
The uID still set to "-1" unsigned, written with the `UINT_MAX` macro, while the command thing can stay zero.